src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -0,0 +1,227 @@
+                return_value=GlobalConfiguration(
+                    recipients_email_digest=["tralala@test.nl", "trblblb@test.nl"]
+                ),
+            ),
+        ):
+            send_email_digest()
+
+        sent_email = mail.outbox[0]
+        submission_occurencies = sent_email.body.count(str(submission.uuid))
+
+        self.assertEqual(
+            sent_email.subject, "[Open Forms] Daily summary of detected problems"
+        )
+        self.assertEqual(
+            sent_email.recipients(), ["tralala@test.nl", "trblblb@test.nl"]
+        )
+        self.assertEqual(submission_occurencies, 1)
+
+    def test_no_email_sent_if_no_logs(self):
+        with patch(
+            "openforms.emails.tasks.GlobalConfiguration.get_solo",
+            return_value=GlobalConfiguration(
+                recipients_email_digest=["tralala@test.nl"]
+            ),
+        ):
+            send_email_digest()
+
+        self.assertEqual(0, len(mail.outbox))
+
+    def test_no_email_sent_if_no_recipients(self):
+        submission = SubmissionFactory.create()
+
+        with freeze_time("2023-01-02T12:30:00+01:00"):
+            logevent.email_status_change(
+                submission,
+                event="registration",
+                status=Message.STATUS_FAILED,
+                status_label="Failed",
+                include_in_daily_digest=True,
+            )
+
+        with (
+            freeze_time("2023-01-03T01:00:00+01:00"),
+            patch(
+                "openforms.emails.tasks.GlobalConfiguration.get_solo",
+                return_value=GlobalConfiguration(recipients_email_digest=[]),
+            ),
+        ):
+            send_email_digest()
+
+        self.assertEqual(0, len(mail.outbox))
+
+    @override_settings(LANGUAGE_CODE="en")
+    def test_email_sent_when_there_are_failures(self):
+        """Integration test for all the possible failures
+
+        - failed_emails
+        - failed_registrations
+        - failed_prefill_plugins
+        - broken_configurations
+        """
+        form = FormFactory.create()
+        submission = SubmissionFactory.create(
+            form=form, registration_status=RegistrationStatuses.failed
+        )
+
+        with freeze_time("2023-01-03T01:00:00+01:00"), patch(
+            "openforms.emails.tasks.GlobalConfiguration.get_solo",
+            return_value=GlobalConfiguration(
+                recipients_email_digest=["tralala@test.nl"]
+            ),
+        ):
+            with self.subTest("failed email"):
+                with freeze_time("2023-01-02T12:30:00+01:00"):
+                    logevent.email_status_change(
+                        submission,
+                        event="registration",
+                        status=Message.STATUS_FAILED,
+                        status_label="Failed",
+                        include_in_daily_digest=True,
+                    )
+
+                send_email_digest()
```

Comment:
I'd set up everything in the beginning of the test, then send the digest once and then make assertions:

```
create form and failed submission
create email status change (failure)
create failed registration log
create failed prefill log

send_email_digest()

assert outgoing email with correct recipient(s)
sent_email = mail.outbox[-1]

with self.subTest("failed registrations")
    assert failed registration

with self.subTest(...)
    assert other thing ...
```